Bottom: 0132521337e539822377263514ee0cea100df275
Top:    cbd0049615c44215ff4ed0e8f36f5f6055cbcb65
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-09-04 16:45:45 +0800

Refresh of vhost-split-out-ring-head

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index 99ad23a..d587e0f 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -191,7 +191,7 @@ int vhost_get_vq_desc(struct vhost_virtqueue *,
 		      struct iovec iov[], unsigned int iov_count,
 		      unsigned int *out_num, unsigned int *in_num,
 		      struct vhost_log *log, unsigned int *log_num);
-int __vhost_get_vq_desc(struct vhost_virtqueue *,
+int __vhost_get_vq_desc(struct vhost_virtqueue *vq,
 			struct iovec iov[], unsigned int iov_count,
 			unsigned int *out_num, unsigned int *in_num,
 			struct vhost_log *log, unsigned int *log_num,
